samples/excel/89-preview-apis/chart-formatting.yaml

```diff
@@ -0,0 +1,328 @@
+id: excel-chart-formatting
+name: Chart - Formatting
+description: Format labels and lines of a slope chart.
+host: EXCEL
+api_set:
+    ExcelAPI: 1.8
+script:
+    content: |+
+        $("#addSampleData").click(() => tryCatch(addSampleData));
+        $("#create-chart").click(() => tryCatch(createChart));
+        $("#format-chart").click(() => tryCatch(formatChart));
+        $("#show-datalabels").click(() => tryCatch(dataLabels));
+        $("#change-color").click(() => tryCatch(changeColor));
+        $("#clear").click(() => tryCatch(clearChart));
+
+        async function createChart() {
+            await Excel.run(async (context) => {
+                var worksheet = context.workbook.worksheets.getActiveWorksheet();
+                var chart = worksheet.charts.add(Excel.ChartType.lineMarkers, worksheet.getRange("B6:G15"), Excel.ChartSeriesBy.rows);
+                chart.axes.categoryAxis.setCategoryNames(worksheet.getRange("C5:G5"));
+                chart.name = "SlopeChart";
+
+                // place chart below sample data
+                chart.top = 275;
+                chart.left = 50;
+                chart.height = 600;
+                chart.width = 850;
+                
+                chart.title.text = "GDP Per Capita , 1980 to 2017";
+                chart.legend.position = "Bottom";
+
+                worksheet.onChanged.add(onDataChanged);
+            });
```

Comment:
Its a good practice to always have one last `await context.sync()` at the end of an `Excel.run`, even though the infrastructure will do this for you if you don't. It makes it clear to someone reading the code that there is a sync at that point. There's no danger of a redundant sync because the infrastructure will not call its own sync if the last line of the `run `calls it explicitly.